test/github-issues/issue-207.test.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+const t = require('tap')
+const test = t.test
+const Fastify = require('fastify')
+const rateLimit = require('../../index')
+
+test('issue #207 - when continueExceeding is true and the store is local then it should reset the rate-limit', async t => {
+  const sleep = (ms) => new Promise((resolve) => setTimeout(resolve, ms))
```

Comment:
This is not very good as it will make the test longer, have any idea of how to solve this? is there some utility in `tap` for _updating_ time?

Reply:
We used https://sinonjs.org/releases/latest/fake-timers/ in others repos